src/main/resources/config.yml

```diff
@@ -13,6 +13,8 @@ fireball:
 tntminecart:
   log: true
   disable-when-target-not-found: true
+bed:
+  log: true
```

Comment:
Is it possible for bed to blow up without a player (target)?

Reply:
I'm not sure, I've just googled it and haven't found anything that could make the bed blow up, maybe it'll explode when /reload'ing in the tick between clicking and exploding. But that's impossible for players to predict reloading, so that's enough.